R/view_palette.R

```diff
@@ -19,6 +19,8 @@ view_palette <- function(palette = palette_urbn_main) {
 
   color_palette <- unname(rev(palette))
 
+  print(paste0("c(", paste(color_palette, collapse = ", "), ")"))
```

Comment:
Is the goal here to have something to copy-paste into an R session? The functions returns:

```r
"c(#db2b27, #5c5859, #55b748, #ec008b, #d2d2d2, #000000, #fdbf11, #1696d2)"
```
 for me - so if so, I think this needs tweaking to be a valid copy/paste block.